java401_code_challenges/src/main/java/java401_code_challenges/hashtable/HashTable.java

```diff
@@ -2,31 +2,31 @@
 
 public class HashTable<T>{
 
-    private int bucketSize = 100;
+    private int numberOfBuckets = 100;
 
-    public Linkedlist[] bucket = new Linkedlist[bucketSize];
+    public Linkedlist[] bucketsArray = new Linkedlist[numberOfBuckets];
 
     public HashTable(){
-        for(int i = 0; i < bucket.length; i++){
-            bucket[i] = new Linkedlist();
+        for(int i = 0; i < bucketsArray.length; i++){
+            bucketsArray[i] = new Linkedlist();
         }
     }
 
 
     public void add(String key, T value){
-        int index = getHash(key)% bucketSize;
-        bucket[index].appendToLinkedList(key, value);
+        int index = getHash(key)% numberOfBuckets;
+        bucketsArray[index].appendToLinkedList(key, value);
```

Comment:
What should be done if a duplicate key is added? Should it be ignored? Overwritten? You are putting it into the list, but it is inaccessible. This is an edge case that needs handled